manifests/k/KOOK/KOOK/0.64.0/KOOK.KOOK.locale.en-US.yaml

```diff
@@ -0,0 +1,46 @@
+# Created with YamlCreate.ps1 v2.2.0 using InputObject 🤖 $debug=QUSU.CRLF..7-2-9
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: KOOK.KOOK
+PackageVersion: 0.64.0
+PackageLocale: en-US
+Publisher: KOOK
+PublisherUrl: https://www.kookapp.cn/
+PublisherSupportUrl: https://help.kookapp.cn/
+PrivacyUrl: https://www.kookapp.cn/privacy.html
+Author: Beijing Xiaoyaoyixia Technology Co., Ltd
+PackageName: KOOK
+PackageUrl: https://www.kookapp.cn/
+License: Proprietary
+LicenseUrl: https://www.kookapp.cn/protocol.html
+Copyright: Copyright © 2023 KOOK
+# CopyrightUrl:
+ShortDescription: An easy-to-use voice chatting tool
+Description: KOOK is a free and ad-free voice chatting tool. It is available on PC, Android, iOS and web, so you can chat anywhere. There are various themes of servers on KOOK. You can also create your own servers with themes of gaming, music, reading, foreign languages, etc. according to your preferences, and invite your friends to join in chatting.
+Moniker: kaiheila
+Tags:
+- chat
+- talk
+- voice
+# Agreements:
+ReleaseNotes: |-
+  2023.02.24更新记录（v0.64.0）
+  功能新增：
+  ● 服务器角色颜色支持渐变，让大家的昵称显示不再单调。
+  ● 服务器助力权益新增国际专线，可获得更优质更稳定的国内外互联通话质量。
+
+  功能优化：
+  ● 现在修改密码会强制退出所有登录的设备。
+  ● 根据小伙伴的建议，我们增加了服务和道具快要过期的提醒。
+  ● 我们小小地优化了一下屏幕分享的观看体验，希望你用的开心～
+  ● 我们优化了音量条的样式，并且语音时可以在麦克风图标上实时看到自己说话音量大小。
+
+  问题修复：
+  ● 修复了部分电脑在玩游戏时，语音质量会降低的问题。
+  ● 修复了一些其他的已知问题。
```

Comment:
```suggestion
  2023.02.24 update record (v0.64.0)
  New features：
  ● The server role color supports gradients, so that everyone's nickname display is no longer monotonous.
  ● The new international private line is added to the server assistance rights, which can obtain better and more stable domestic and foreign interconnection call quality.

  Function optimization:
  ● Changing your password now forces you to log out of all logged in devices.
  ● According to the suggestions of our friends, we have added reminders that services and items are about to expire.
  ● We have slightly optimized the viewing experience of screen sharing, hope you enjoy it!
  ●We have optimized the style of the volume bar, and you can see your speaking volume in real time on the microphone icon when speaking.

  Bug fixes：
  ● Fixed an issue where the voice quality would decrease on some computers when playing games.
  ● Fixed some other known issues.
```